dagmc_stats/DagmcStats.py

```diff
@@ -40,3 +39,24 @@ def __init__(self, filename, populate=False):
         self._vol_data = pd.DataFrame(
             columns=['vol_eh', 'surf_per_vol', 'coarseness'])
         self._vol_data = self._vol_data.set_index('vol_eh')
+        
+        self.entity_types = [types.MBVERTEX, types.MBTRI, types.MBENTITYSET]
+        self.native_ranges = {}
+        self.__set_native_ranges()
+
+    def __set_native_ranges(self):
+        """Set the class native_ranges variable to a dictionary with MOAB
+        ranges for each of the requested entity types
+
+        inputs
+        ------
+        none
+
+        outputs
+        -------
+        native_ranges : a dictionary with one entry for each entity type that is a
+                        Range of handles to that type
+        """
+        for entity_type in self.entity_types:
```

Comment:
It may make more sense to explicitly refer to the class variable here?  It appears that both will work, but `DagmcStats.entity_types` is accessible in various contexts/scopes, and it may be more logical to use it that way throughout.

```suggestion
        for entity_type in DagmcStats.entity_types:
```